Library/Homebrew/cmd/install.rb

```diff
@@ -40,20 +40,28 @@ def install
       end
 
       ARGV.formulae.each do |f|
-        # Building head-only without --HEAD is an error
-        if not ARGV.build_head? and f.stable.nil?
+        # head-only without --HEAD is an error
+        if not ARGV.build_head? and f.stable.nil? and f.devel.nil?
           raise <<-EOS.undent
           #{f.name} is a head-only formula
           Install with `brew install --HEAD #{f.name}`
           EOS
         end
 
-        # Building stable-only with --HEAD is an error
+        # devel-only without --devel is an error
+        if not ARGV.build_devel? and f.stable.nil?
```

Comment:
Yup. Makes sense since we haven't banned the use of ` head ` within ` devel-only `. Pushing now :+1: